src/main/config/app.ts

```diff
@@ -0,0 +1,25 @@
+import { CreatePokemonLoaderService } from '@/data/services';
+import { AppDataSource } from '@/infra/data-sources/typeorm'
+import { setupRoutes } from '@/main/config/routes'
+import express from 'express';
+import { TypeOrmPokemonRepository } from '@/infra/repositories';
+
+AppDataSource.initialize()
+.then(async () => {
+    console.log("Data Source has been initialized!")
+
+    const pokemonRepository = new TypeOrmPokemonRepository();
+
+    const createPokemonLoaderService = new CreatePokemonLoaderService(pokemonRepository);
+
+    await createPokemonLoaderService.load();
```

Comment:
Colocar o import da planilha hard coded acaba por forçar uma consulta desnecessária do banco de dados a cada vez que a aplicação é iniciada, pouco mais de 800 linhas são rápidas pra ler, mas num banco maior, o custo é muito mais alto.